addons/mail/models/mail_mail.py

```diff
@@ -336,6 +337,17 @@ def _send_prepare_values(self, partner=None):
         """
         self.ensure_one()
         body = self._send_prepare_body()
+        if partner and 'unfollow_placeholder_start' in body and \
```

Comment:
can you wrap the condition in `()` ? (so no need for `\`)

Reply:
yes forgot about that trick, thanks.